python/cugraph/cugraph/testing/mg_utils.py

```diff
@@ -77,3 +86,185 @@ def stop_dask_client(client, cluster=None):
     if cluster:
         cluster.close()
     print("\ndask_client closed.")
+
+
+def enable_spilling():
+    import cudf
+
+    cudf.set_option("spill", True)
+
+
+def generate_edgelist(
+    scale,
+    edgefactor,
+    seed=None,
+    unweighted=False,
+):
+    """
+    Returns a dask_cudf DataFrame created using the R-MAT graph generator.
+
+    The resulting graph is weighted with random values of a uniform distribution
+    from the interval [0, 1)
+
+    scale is used to determine the number of vertices to be generated (num_verts
+    = 2^scale), which is also used to determine the data type for the vertex ID
+    values in the DataFrame.
+
+    edgefactor determies the number of edges (num_edges = num_edges*edgefactor)
+
+    seed, if specified, will be used as the seed to the RNG.
+
+    unweighted determines if the resulting edgelist will have randomly-generated
+    weightes ranging in value between [0, 1). If True, an edgelist with only 2
+    columns is returned.
+    """
+    ddf = rmat(
+        scale,
+        (2**scale) * edgefactor,
+        0.57,  # from Graph500
+        0.19,  # from Graph500
+        0.19,  # from Graph500
+        seed or 42,
+        clip_and_flip=False,
+        scramble_vertex_ids=True,
+        create_using=None,  # return edgelist instead of Graph instance
+        mg=True,
```

Comment:
Should this be a parameter to the function to allow for both SG `cudf` and MG `dask_cudf` return types too?

Reply:
Resolved by https://github.com/rapidsai/cugraph/commit/005b698413ad4f3b849461831175ec75f04533f1